elements/rh-blockquote/docs/10-style.md

```diff
@@ -1,100 +1,200 @@
+           alt="Image of two blockquotes, both with red title text and white header text",
+           src="../blockquote-title-heading-theme-dark.png" %}
 
-<hr style="margin-block:var(--rh-space-5xl);">
+| Element      | Light theme | Dark theme |
+|--------------|-------------|------------|
+| Title text   | #EE0000     | #FF542E    |
+| Heading text | #151515     | #FFFFFF    |
 
-{% section %}
-## Responsive design
+{.full-width}
 
-A blockquote can be used on large and small screens, but content will get 
-longer as space reduces.
+## Configuration
 
-### Breakpoints
-When breakpoints get smaller, a left-aligned blockquote will stay left-aligned, 
-same with a center-aligned blockquote. If a left-aligned blockquote is used with 
-components, they drop underneath the quote and all elements are then organized 
-in one column.
+The base elements in both sizes are stacked and left aligned by default, but they can be vertically centered if necessary.
 
-### Desktop
+{% example palette="light",
+           width=872,
+           class="centered",
+           alt="Image of four blockquotes, two are left aligned and two are vertically centered, the quote icon is 20px tall",
+           src="../blockquote-configuration.png" %}
+
+### Order
+A blockquote was designed to be read from top to bottom. If certain optional elements are included, the order will change.
+
+{% example palette="light",
+           width=693,
+           class="centered",
+           alt="Image of a blockquote with numbers 1 - 4 on the right side going from top to bottom",
+           src="../blockquote-configuration.png" %}
+1. Logo or text (always ordered first if included)
+2. Quotate icon (always included and ordered first if there is no logo or text)
+3. Quotation text (always ordered after the quote icon)
+4. Citation text (always ordered last)
+
+### Citation text
+Citation text has specific styles applied to it.
+
+{% example palette="light",
+           width=349,
+           class="centered",
+           alt="Image of three citation text examples",
+           src="../blockquote-configuration-citation.png" %}
 
-<div style="display:flex;flex-direction:column;gap:var(--rh-space-2xl);margin-block:var(--rh-space-2xl);">
-  <img alt="blockquote on desktop" src="{{ '../blockquote-breakpoint-1.svg' | url }}">
-  <img alt="Centered blockquote on desktop" src="{{ '../blockquote-breakpoint-2.svg' | url }}">
-  <img alt="blockquote on tablet" src="{{ '../blockquote-breakpoint-3.svg' | url }}">
-  <img alt="blockquote on tablet centered" src="{{ '../blockquote-breakpoint-4.svg' | url }}">
-</div>
+| Property                            | Current value |
+|-------------------------------------|---------------|
+| Font family - all text              | Red Hat Text  |
+| Font weight - name                  | Medium        |
+| Font weight - job title and company | Regular       |
+| Font style - company                | Italic        |
 
-### Tablet
+ {.full-width} 
```

Comment:
@coreyvickery do you want this one hidden too?

Reply:
@zeroedin Yes please.